app/assets/stylesheets/components/shared/c-dropdown-selector.scss

```diff
@@ -21,6 +22,7 @@
     opacity: 0;
     z-index: 15;
     pointer-events: none;
+    overflow-y: auto;
```

Comment:
Don't forget this property too: `-webkit-overflow-scrolling`. It's needed for iOS devices to have a smooth scrolling behaviour.